src/components/Search/Search.jsx

```diff
@@ -16,7 +16,7 @@ const Search = ({ initialQuery, placeholder, onSearch }) => {
   };
 
   return (
-    <form onSubmit={handleSubmit} className="search-form">
+    <form onSubmit={handleSubmit} className={styles.searchForm}>
       <div>{Search_Header_Text}</div>
       <input
         data-testid="search-input"
```

Comment:
Try to add prop `name="query"` to the `input` element and submit in "Disabled JavaScript" mode. Here is more info:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/form